### DISCUSSION TITLE
How to set colors in a extra file (example dead link)

### DISCUSSION THREAD

**Discussion by juridiener** [wezterm/wezterm#4452] [Q&A]
### What Operating System(s) are you running on?

macOS

### WezTerm version

20230712-072601-f4abf8fd

### Ask your question!

Hello on the docs is an example, but the link is dead. https://github.com/wez/wezterm/blob/main/assets/colors/Builtin%20Dark.toml

I created a colors directory and put a .toml file in it.
This is the Content: https://github.com/EdenEast/nightfox.nvim/blob/main/extra/dawnfox/nightfox_wezterm.toml

But how to use this that the colors are applied to wezterm? 

And another question when i try to use a color scheme for example dawnfox like so:
config.color_scheme = 'dawnfox'

Should this change everything? So background color, tabbar background color and so one...?

**Comment by marchank0** [comment]
@juridiener you need to put in your config to load colorscheme from toml

```lua
local schemes = {
	"/Users/ama/.config/wezterm/colors/rose-pine-dawn.toml",
	"/Users/ama/.config/wezterm/colors/rose-pine-moon.toml",
}
for _, scheme in ipairs(schemes) do
	wez.color.load_scheme(scheme)
end
```

**Comment by wez** [comment]
https://wezfurlong.org/wezterm/config/appearance.html#defining-a-color-scheme-in-a-separate-file mentions the `color_scheme_dirs` configuration option and the suggested default location of `$HOME/.config/wezterm/colors`.

Wezterm scans the color scheme directories and loads the schemes, so you should be able to just do `config.color_scheme = 'dawnfox'` to activate https://github.com/EdenEast/nightfox.nvim/blob/df75a6a94910ae47854341d6b5a6fd483192c0eb/extra/dawnfox/wezterm.toml#L2 based on its name metadata.

**Reply by marchank0** [reply]
> `$HOME/.config/wezterm/colors`.

thanks! don't know how I missed that
